LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2010-2011 Stripe (http://stripe.com)
+Copyright (c) 2010-2018 Stripe (http://stripe.com)
```

Comment:
This one hadn't been updated in a while ;)

Reply:
lol. Maybe another seven years until the next time too ...